src/docs/guides/microsoft-entra-id-integration/README.md

```diff
@@ -1,4 +1,4 @@
-# Use Azure AD as an external identity provider
+# Use Microsoft Entra ID / Azure AD as an external identity provider
```

Comment:
Run the docs locally, see https://docs.orchardcore.net/en/latest/docs/topics/docs-contributions/, to see if anything got broken. There's this one at least:

![image](https://github.com/OrchardCMS/OrchardCore/assets/1976647/7c85485d-df92-471c-8e1c-be2fc0d8762f)